packages/layout/src/components/__tests__/actuary.spec.tsx

```diff
@@ -201,7 +206,9 @@ describe('Actuary Card', () => {
 		test('save and close', async () => {
 			await act(async () => {
 				findByText('Save and close').click();
-				const results = await axe(component);
+				const results = await axe(component, {
+					rules: disableHeadingOrder,
```

Comment:
I'm curious as to why this needs to be disabled? Is it just saying "you can't start with h3?"

Reply:
it happens after the section title was converted to H5. 
For some reason, jest-axe fails when reading the H5, saying "Heading levels should only increase by one (heading-order)" even though we have H3 for the main heading and H4 for the section button.
It might be because the H4 is wrapped with a div at the level of the H5, and a possible solution could be wrapping the H5 with a div too, but I don't think is good adding unnecessary divs.

Reply:
rule removed and tests pass correctly with the section heading now having conditional heading H4/H5